tools/site_admin/copy_pages.php

```diff
@@ -233,6 +233,14 @@ function do_stuff( $projectid_, $from_image_, $page_name_handling,
 
     foreach ( array( 'from', 'to' ) as $which )
     {
+        $projectid = $projectid_[$which];
+        $project = new Project($projectid);
+
+        if(!$project->is_utf8)
+        {
+            die("Project table $projectid is not UTF-8.");
```

Comment:
I'm not terribly concerned about that siteadmin scripts be translatable, but this should probably have gettext() markup to be consistent? Ditto above question about back-link.